src/main/java/calculator/Operator.java

```diff
@@ -0,0 +1,30 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.function.BiFunction;
+
+public enum Operator {
+	PLUS("+", (firstNumber, secondNumber) -> firstNumber + secondNumber),
+	MINUS("-", (firstNumber, secondNumber) -> firstNumber - secondNumber),
+	TIMES("*", (firstNumber, secondNumber) -> firstNumber * secondNumber),
+	DIVIDE("/", (firstNumber, secondNumber) -> firstNumber / secondNumber);
+
+	private final String operator;
+	private final BiFunction<Integer, Integer, Integer> function;
+
+	Operator(String operator, BiFunction<Integer, Integer, Integer> function) {
+		this.operator = operator;
+		this.function = function;
+	}
+
+	public Integer calculate(Integer firstNumber, Integer secondNumber) {
```

Comment:
파라미터로 Integer를 사용하면 null을 전달 받을 위험이 있습니다
int로 제한하는게 좋겠네요
```suggestion
	public int calculate(int firstNumber, int secondNumber) {
```